osmhelper/osm_helper.py

```diff
@@ -19,8 +19,22 @@ class OsmHelper(object):
     def __init__(self, config):
         self.config = config
 
+        # bbox from config file for querying
+        self.bbox = (str(config['query']['bbox']["s"]) + "," +
+                     str(config['query']['bbox']["w"]) + "," +
+                     str(config['query']['bbox']["n"]) + "," +
+                     str(config['query']['bbox']["e"]))
+
+        # tags from config file for querying
+        self.tags = ''
+        for key, value in config["query"].get("tags", {}).iteritems():
+            self.tags += str('["'+key+'" = "'+value+'"]')
```

Comment:
Just a style nipick: PEP8 asks for spaces around the operators and I agree that it is more readable this way.